models/portfolioOverlap.js

```diff
@@ -0,0 +1,100 @@
+const fs = require('fs')
+const path = require('path')
+const { Sequelize, DataTypes } = require('sequelize')
+const basename = path.basename(module.filename)
+const config = require(__dirname + '/../config/db.json').mysql
+const sequelize = new Sequelize(config.database, null, null, config)
+
+sequelize
+  .authenticate()
+  .then(() => {
+    console.log('Connection has been established successfully....')
+  })
+  .catch((error) => {
+    console.error('Unable to connect to the database: ', error)
+  })
+
+const scheme = sequelize.define('scheme', {
+    fundid: {
+      type: DataTypes.INTEGER,
+      allowNull: false,
+      primaryKey: true,
+    },
+    schid: {
+      type: DataTypes.INTEGER,
+      allowNull: false,
+    },
+    name: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+    fsid: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    }
+  },
+  {
+    tableName: 'schemes'
+  }
+)
+
+const schemeholding = sequelize.define('schemeHolding',{
+    holdingid: {
+      type: DataTypes.INTEGER,
+      allowNull: false,
+      primaryKey: true,
+    },
+    holdings: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+    netAsset: {
+      type: DataTypes.FLOAT,
+      allowNull: true,
+    },
+    fsid: {
+      type: DataTypes.INTEGER,
+      allowNull: true,
+    },
+  },
+  {
+    tableName: 'schemeHolding'
+  }
+)
+
+const schemedetail = sequelize.define('schemeDetail',{
```

Comment:
give space after 
('schemeDetail', {
'schemeHolding', {


correct the same everywhere